pull_request_template.md

```diff
@@ -13,6 +13,14 @@ What this PR includes
 <!-- Brief description of what was done to meet the PR goal. -->
 
 
+How this change is ready for production release
+-----------------------------------------------
+<!-- Required. Select one of these checkboxes, otherwise merging will be blocked. -->
+<!-- e2e-testing:start -->
+- [ ] End-to-end tests ran and passed, or
+- [ ] [Release exemption](https://energyhub.slack.com/archives/C098SCVCP55) granted: [link here](), or
+- [ ] Not releasing to production
+<!-- e2e-testing:end -->
```

Comment:
Should there be an "N/A" option for things like documentation/comment only changes, CICD changes, test-only changes, and so forth?